core/src/md5_stubs.c

```diff
@@ -11,6 +11,16 @@
 #if __GNUC__ < 8
 #pragma GCC diagnostic ignored "-pedantic"
 #endif
+
+// Copied from caml/misc.h
+#ifdef __GNUC__
+#define CAMLlikely(e)   __builtin_expect(!!(e), 1)
+#define CAMLunlikely(e) __builtin_expect(!!(e), 0)
+#else
+#define CAMLlikely(e) (e)
+#define CAMLunlikely(e) (e)
+#endif
```

Comment:
C isn't my strong suit, and I couldn't figure out a way to avoid the compilation error here without copying in the `#define` from `runtime/caml/misc.h`.

Reply:
why not move `CAML_INTERNALS` up like i’ve done in https://github.com/janestreet/core/pull/154/files#diff-a4dd01283961659d7b016080309ef8ab6ea68f01f4052f0c16eb5c6398e055e1 ?

Reply:
Ah! Thank you! I haven't written any C bindings in OCaml that use `CAML_INTERNALS` so hadn't encountered this before!